packages/build/src/build-variant.ts

```diff
@@ -6,6 +6,7 @@
  */
 enum BuildVariant {
   Windows = 'win32',
+  WindowsMSI = 'win32msi',
```

Comment:
Mh should add this to `ALL_BUILD_VARIANTS` just immediately below? 🤔 Or are there reasons we would not want it in there - then I'd have to think of a better variable name for that array...

There are tests checking that array so everything is accounted for - but I should probably extend them so that they should already fail when you modify the enum without adapting the array :D

Once that is in there `run-draft` will automatically pick it up and upload to GitHub / DownloadCenter.

Reply:
Well... I'll add it and we see what happens 🤷‍♀️